src/App.jsx

```diff
@@ -14,57 +16,53 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [isSelected, setSelected] = useState('Jam');
```

Comment:
isSelected - sounds like boolean var

better to rename to selectedGood or something like that